tests/test_api.py

```diff
@@ -979,6 +979,44 @@ def test_build_image(self):
         img = req.json['spec']['strategy']['customStrategy']['from']['name']
         assert img == build_image
 
+    def test_build_image_with_platform_node(self):
+        build_image = 'registry.example.com/buildroot:2.0'
+        with NamedTemporaryFile(mode='wt') as fp:
+            fp.write(dedent("""\
+                [general]
+                build_json_dir = {build_json_dir}
+                [default]
+                openshift_url = /
+                sources_command = /bin/true
+                vendor = Example, Inc
+                registry_uri = registry.example.com
+                build_host = localhost
+                authoritative_registry = localhost
+                distribution_scope = private
+                platform = breakfast
+                node_selector.breakfast = bacon.com
```

Comment:
A node selector is a set of key=value pairs, like `node_selector.breakfast = spam=eggs,ham=green`, and expect to have a node selector with key 'spam' having value 'eggs' and key 'ham' having value 'green'.